orchestrator-bundle/orc8r-orchestrator-operator/tests/unit/test_charm.py

```diff
@@ -193,6 +194,37 @@ def test_given_relations_ready_and_container_can_connect_when_on_pebble_ready_ev
         ]
         self.assertIn(call_command, args)
 
+    @patch("ops.model.Container.exists")
+    @patch("ops.model.Container.exec", new_callable=Mock)
+    @patch("ops.model.Container.push")
+    def test_given_relations_ready_and_container_can_connect_and_unit_is_not_leader_when_on_pebble_ready_event_then_orchestrator_admin_user_is_created(  # noqa: E501
```

Comment:
Didn't you mean to say `is_not_created`?
```suggestion
    def test_given_relations_ready_and_container_can_connect_and_unit_is_not_leader_when_on_pebble_ready_event_then_orchestrator_admin_user_is_not_created(  # noqa: E501
```

Reply:
Yes, good catch!